backend/scc.py

```diff
@@ -0,0 +1,40 @@
+from datetime import date, datetime, timedelta
+import pyodbc
+import pandas as pd
+import requests
+
+
+
+# URL of the file you want to download
+file_url = 'https://cmsprovider.cahwnet.gov/prv/scc.xls'
+
+# Local path to save the downloaded file
+file_path = 'D:/Python/scc.xls'
```

Comment:
add a todo here as well because this path will be on the server and feed from a variable